shared/src/mobileMain/kotlin/dev/johnoreilly/confetti/SessionsViewModel.kt

```diff
@@ -147,6 +156,25 @@ open class SessionsViewModel : KMMViewModel(), KoinComponent {
 
     suspend fun refresh() = refresh(false)
 
+    fun onSearch(search: String) {
+        _search.value = search
+    }
+
+    private fun filterSessions(uiState: SessionsUiState, filter: String): SessionsUiState {
+        return if (filter.isNotBlank() && uiState is SessionsUiState.Success) {
+            val newSessions = uiState.sessionsByStartTimeList.map { outerMap ->
+                outerMap.mapValues { (_, value) ->
+                    value.filter { session ->
+                        session.title.contains(filter, ignoreCase = true)
```

Comment:
Might be worth searching the description as well?

Reply:
Done. I have included the title, description, room, and speakers.

Reply:
Thanks!